.gitignore

```diff
@@ -3,3 +3,4 @@
 /checkver*
 .idea
 *.iml
+.vscode
```

Comment:
Please don't modify .gitignore

Reply:
.idea is here, I think it makes sense to also have an ignore for .vscode, since it a very popular editor

Reply:
For something similar to that, you would have to create a new PR.

Reply:
created https://github.com/ScoopInstaller/Main/pull/2898